### PR TITLE
My custom Polaris keymap

### PR DESCRIPTION
**Pull request by Endemoniada**
## Description

This is my custom take on the Polaris keymap. It's a mashup between "standard" Swedish ISO and physical ANSI/HHKB layout, creating what I feel is a pleasant mix of features.

Core features include making space for the ÅÖÄ keys by moving the other keys up to the split-backspace position, and having navigation on IJKL with LayerTap() on the Tab key.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* N/A

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ai03/polaris/keymaps/mekberg/config.h [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#pragma once
```
Comment (drashna) [inline keyboards/ai03/polaris/keymaps/mekberg/config.h:1]: Could you add a GPL2+ license header, here?

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/ai03/polaris/keymaps/mekberg/config.h (hunk 1 above)
Comment (Endemoniada) [reply keyboards/ai03/polaris/keymaps/mekberg/config.h:18]: Sure, like that? I copied over the one from the original Polaris files, so they're the same.

File: keyboards/ai03/polaris/keymaps/mekberg/readme.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Fn key layer is focused mostly on keyboard firmware features (like RGB) as well as some of the necessary alternate functions. It also hold media controls, F-keys and Reset.
+
+Left Tab-Hold layer is focused on nav cluster functionality, and turns Backspace into Delete (and moves Backspace up into the top right 1u position).
+
+
+# Build instructions
+
+To simply build the firmware file: `make clean && make ai03/polaris:mekberg`
+
+To build and immediately flash: `make clean && make ai03/polaris:mekberg:dfu`
```
Comment (fauxpark) [inline keyboards/ai03/polaris/keymaps/mekberg/readme.md:14]:
```suggestion
To build and immediately flash: `make clean && make ai03/polaris:mekberg:flash`
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/ai03/polaris/keymaps/mekberg/readme.md (hunk 2 above)
Comment (Endemoniada) [reply keyboards/ai03/polaris/keymaps/mekberg/readme.md:14]: Cool! I actually had no idea about this, but I just verified it works and I'll make the change. Thanks!

File: keyboards/ai03/polaris/keymaps/mekberg/config.h (hunk 1 above)
Comment (drashna) [reply keyboards/ai03/polaris/keymaps/mekberg/config.h:18]: Awesome! Thanks!
And you can add your name there too, if you want.

File: keyboards/ai03/polaris/keymaps/mekberg/readme.md (hunk 2 above)
Comment (drashna) [reply keyboards/ai03/polaris/keymaps/mekberg/readme.md]: Yup, the flash target looks at the bootloader config, and attempts to flash with the "correct" method. 

Also, you can use `make clean ai03/polaris:mekberg:flash`, as well.

**Review by drashna — APPROVED** [review]

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**